unicodetools/src/main/java/org/unicode/text/UCD/UCD_Names.java

```diff
@@ -1220,7 +1220,18 @@ public final class UCD_Names implements UCD_Types {
         "Hanifi_Rohingya_Kinna_Ya",
         // Unicode 14
         "THIN_YEH",
-        "VERTICAL_TAIL"
+        "VERTICAL_TAIL",
+        // Unicode n > 17
```

Comment:
I don't know the intention of the notation here, but shouldn't it either say "> 16" or ">= 17"?

Reply:
Certainly > 16, but probably not = 17 either, see the latest RMG minutes.
Not that this comment matters much, it will change to whatever version this ends up going in.

Reply:
> see the latest RMG minutes.

Where is that? I'm not in RMG.

Reply:
(See email.)